zuul-sample/src/main/java/com/netflix/zuul/sample/ZuulSampleModule.java

```diff
@@ -45,6 +47,13 @@
 public class ZuulSampleModule extends AbstractModule {
     @Override
     protected void configure() {
+        try {
+          ConfigurationManager.loadCascadedPropertiesFromResources("application");
+        } catch (Exception ex) {
+
```

Comment:
this should re throw as a RuntimeException

Reply:
Ooops I mentally noted I'd need to do something with that catch block when I left it empty. Thanks for catching that. Updated.